### PR TITLE
chore: Playwright config nits

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -30,12 +29,13 @@ export default {
...
-			: `cross-env PORT=${PORT} npm run dev`,
+		command: process.env.CI ? 'npm run start:mocks' : 'npm run dev',
 		port: Number(PORT),
-		reuseExistingServer: true,
+		reuseExistingServer: !process.env.CI,
 		stdout: 'pipe',
 		stderr: 'pipe',
+		env: {
+			PORT,
+		},
```
Comment (kentcdodds) [inline playwright.config.ts:39]: This is nice!

File: tests/playwright-utils.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ export const test = base.extend<{
 }>({
 	login: [
 		async ({ page, baseURL }, use) => {
-			use(user => loginPage({ page, baseURL, user }))
+			await use(user => loginPage({ page, baseURL, user }))
```
Comment (kentcdodds) [inline tests/playwright-utils.ts:38]: Ah, yeah 👍

File: playwright.config.ts [hunk 3]
```diff
@@ -30,12 +29,13 @@ export default {
 	],
 
 	webServer: {
-		command: process.env.CI
-			? `cross-env PORT=${PORT} npm run start:mocks`
-			: `cross-env PORT=${PORT} npm run dev`,
+		command: process.env.CI ? 'npm run start:mocks' : 'npm run dev',
 		port: Number(PORT),
-		reuseExistingServer: true,
+		reuseExistingServer: !process.env.CI,
```
[the remark this answers is not in the document]
Comment (mxschmitt) [reply playwright.config.ts:34]: Not necessary, but we recommend it for the use-case that you only have locally some existing development server running. On the CI you _hopefully_ never do. To catch the case that port 3000 might be used already on the CI, e.g. caused by some zombie processes there, we add this conditional check there.

[the remark this answers is not in the document]
Comment (kentcdodds) [reply playwright.config.ts:34]: Gotcha 👍 Sounds good.

**Pull request merged**